### DISCUSSION TITLE
Can't find the Google Login in the redash open source

### DISCUSSION THREAD

**Discussion by vijay8059** [getredash/redash#6643] [General]
Hi,
Need an help where i am unable to find the enable the Can't find the Google Login in the redash. is there any changes i need to do to set it.
Thanks in Advance.

**Comment by masayuki038** [comment]
Hi @vijay8059 
I haven't tried that. However, it is described in:

- https://redash.io/help/user-guide/users/authentication-options
  - "Google-Login-(OAuth)"

Can you try it?
